modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectEntry/ModalDeleteObjectEntry.tsx

```diff
@@ -0,0 +1,102 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayButton from '@clayui/button';
+import ClayModal from '@clayui/modal';
+import {fetch, openToast} from 'frontend-js-web';
+import React from 'react';
+
+interface ItemData {
+	actions: ItemDataDelete;
+	id: number;
+}
+
+interface ItemDataDeleteProps {
```

Comment:
@Artartur You don't need to add "Props" at the end of this name.

Reply:
Also, we already have the interface `HTTPMethod` that can be used here, so you don't need to declare this one.

Reply:
Done!